device/ospfv3/interface/interface.yaml

```diff
@@ -0,0 +1,179 @@
+          type: string
+          format: ipv4
+          x-field-uid: 3
+
+    Ospfv3Interface.NetworkType:
+      description: >-
+        The OSPFv3 network link type options.
+          - Broadcast
+          - Point to Point
+      type: object
+      properties:
+        choice:
+          type: string
+          default: broadcast
+          x-field-uid: 1
+          x-enum:
+            broadcast:
+              x-field-uid: 1
+            point_to_point:
+              x-field-uid: 2
+        
+    Ospfv3Interface.Capabilities:
+      description: >-
+        Contains OSPFv3 interface capabilities.
+      type: object
+      properties:
+        hello_interval:
+          description: |-
+            The time interval, in seconds, between the Hello packets that
+            the router sends on the interface. Advertised in Hello packets
+            sent out this interface.
+          type: integer
+          format: uint32
+          default: 10
+          maximum: 65535
+          x-field-uid: 1
+        dead_interval:
+          description: |-
+            The time interval in seconds before the router's neighbors will declare
+            it down, when they stop hearing the router's Hello Packets.
+            Advertised in Hello packets sent out this interface.
+          type: integer
+          format: uint32
+          default: 40
+          maximum: 65535
+          x-field-uid: 2
+        link_metric:
+          description: |-
+            The cost of transmitting data on this link.
+          type: integer
+          format: uint32
+          default: 10
+          x-field-uid: 3
+        priority:
+          description: |-
+            The Priority for (Backup) Designated Router election.          	
+            This value is used in Hello packets for the Designated Router (DR) election process.
+            The default is 0, which indicates that the router will not participate in the DR election process.
+          type: integer
+          format: uint32
+          default: 0
+          maximum: 255
+          x-field-uid: 4
+        validate_received_mtu:
+          description: |
+            If this is set to true, then the MTU received from the neighbor during Database (DB) Exchange 
+            will be validated, otherwise it will be ignored.
+          type: boolean
+          default: true
+          x-field-uid: 5
+    
+    Ospfv3Interface.Options:
+      description: |-
+        The Options field is present in OSPFv3 Hello packets, Database Description packets and all LSAs.
+        The Options field enables OSPF routers to support (or not support) optional capabilities, and to
+        communicate their capability level to other OSPF routers (https://datatracker.ietf.org/doc/html/rfc2740#appendix-A.2).
+        When used in Hello packets, the Options field allows a router to reject a neighbor because of a capability mismatch.
+      type: object
+      properties:
+        dc_bit:
```

Comment:
Some of these bits should probably be in the router level in the model , considering multiple interfaces. Need to check RFC once on which bits MUST be same across multiple interfaces and which are truly per interface information.

Reply:
Checked, looks like they belong in interface level only.